src/main/java/UtilsTest/PasswordsTest.java

```diff
@@ -0,0 +1,66 @@
+package UtilsTest;
+
+import Utils.Passwords;
+import Utils.Tuple;
+import jdk.nashorn.internal.ir.annotations.Ignore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PasswordsTest {
+
+    String password;
+
+    @BeforeEach
```

Comment:
The `@BeforeEach` annotated function will run before every test in the class. If there is duplicate code for every test, it should be placed here.